pkg/occlient/occlient.go

```diff
@@ -1193,6 +1160,12 @@ func (c *Client) BootstrapSupervisoredS2I(params CreateArgs, commonObjectMeta me
 	addBootstrapSupervisordInitContainer(&dc, commonObjectMeta.Name)
 	addBootstrapVolume(&dc, commonObjectMeta.Name)
 	addBootstrapVolumeMount(&dc, commonObjectMeta.Name)
+	// only use the deployment Directory volume mount if its being used and
+	// its not a sub directory of src_or_bin_path
+	if s2iPaths.DeploymentDir != "" && !isSubDir("/opt/app-root", s2iPaths.DeploymentDir) {
```

Comment:
`/opt/app-root` shouldn't be here.. variable should be taken from a constant.

Reply:
done